src/service.js

```diff
@@ -14,7 +14,7 @@ module.exports = function commands(robot) {
   async function getUsers(res) {
     const { room } = res.message;
     const users = await repository.getUsers({ room });
-    const limit = 10;
+    const limit = 12;
```

Comment:
Change this to a class variable so that is not repeated  everywhere

Reply:
really? a repeated variable?